src/screens/tenants/Tenants.tsx

```diff
@@ -41,6 +41,8 @@ export default class Tenants extends BaseScreen<Props, State> {
     await super.componentDidMount();
     const tenants = await this.centralServerProvider.getTenants();
     this.setState({ tenants });
+    const createQrCodeTenantVisible = Utils.getParamFromNavigation(this.props.route, 'openQRCode', this.state.createQrCodeTenantVisible);
+    this.setState({createQrCodeTenantVisible})
```

Comment:
No double setState() here, use only one + missing semicolon.

```
this.setState({
  tenants,
  createQrCodeTenantVisible
});
```

Reply:
But boolean intermediate navigation not needed if you can navigate right away in the QR-Code screen